lib/aws/active_job/sqs/configuration.rb

```diff
@@ -87,12 +111,24 @@ class Configuration
 
         def initialize(options = {})
```

Comment:
https://github.com/aws/aws-sessionstore-dynamodb-ruby/blob/main/lib/aws/session_store/dynamo_db/configuration.rb#L101

This style/approach reads a lot cleaner IMO